pull_request_template.md

```diff
@@ -2,11 +2,13 @@
 - [ ] Bugfix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] Styling- no new features
+- [X] Styling- no new features
```

Comment:
Just a heads up - you don't need to edit this locally, this will autofill the PR description box when you make a PR on github.

Reply:
thanks for letting me know